habitat/omnitruck-web/default.toml

```diff
@@ -1,14 +1,7 @@
-port = 80
-app_environment = "production"
-
 [unicorn]
 listen = "/hab/svc/omnitruck/var/unicorn.sock.0"
```

Comment:
⚠️ Bike Shed ⚠️ 

We should be able to use TCP sockets `listen 8080, :tcp_nopush => true` which will allow this all to work as separate containers. I wouldn't spend too much time on this if it doesn't "just work" though.